AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -736,9 +738,12 @@ - (BOOL)checkForUpdatesAllowed {
   // Check if we are not in AppStore or TestFlight environments.
   BOOL environmentOkay = [MSUtility currentAppEnvironment] == MSEnvironmentOther;
 
+  // Check if we are currently in guided access mode. Guided access mode prevents opening update URLs.
+  BOOL noGuidedAccessMode = ![MSGuidedAccessUtil isGuidedAccessEnabled];
```

Comment:
It would be better to use positive variable name here by removing `!`.
```suggestion
  BOOL guidedAccessModeEnabled = [MSGuidedAccessUtil isGuidedAccessEnabled];
```